lib/model/message_list.dart

```diff
@@ -613,6 +619,15 @@ class MessageListView with ChangeNotifier, _MessageSequence {
   Anchor get anchor => _anchor;
   Anchor _anchor;
 
+  @override bool get oneMessagePerBlock => switch (narrow) {
```

Comment:
nit: this feels to me like it belongs after the lifecycle methods, next to _messageVisible and friends